node_modules/app/format_date.js

```diff
@@ -21,8 +21,7 @@ function date_only (d) {
 
 function format_date (d) {
   var date = check_date(d) // allows arg to be date string
-  date.map(d => date_to_string((new Date(d)).setHours(12))) // hack because of weirdness with times, sails and waterline
-  date = date_only(date)
+  date = date_only(date.map(d => date_to_string((new Date(d)).setHours(12)))) // hack because of weirdness with times, sails and waterline
```

Comment:
Cool. I'll merge this, but maybe we should have just had

```js
var date = date_only(check_date(d).map(d => date_to_string((new Date(d)).setHours(12))))
```

In fact we could have

```js
var format_date = compose
  ( map(s => s.split('-').reverse().join('/')
  , date_only
  , map(d => date_to_string((new Date(d)).setHours(12)))
  , check_date
  )
```